File: Newclass.php
<?php
if ($_SERVER['REQUEST_METHOD'] === 'POST') {
  $className = $_POST['class-name'];
  $classPeriod = $_POST['period'];

  // Connect to the database
  $db = new SQLite3('Usertest.db');

  // Prepare the SQL statement to insert the new class
  $stmt = $db->prepare('INSERT INTO classes (name, period) VALUES (:name, :period)');
  $stmt->bindValue(':name', $className);
  $stmt->bindValue(':period', $classPeriod);

  // Execute the SQL statement
  $stmt->execute();

  // Close the database connection
  $db->close();

  // Redirect to the classes page
  header('Location: classes.html');
  exit();
}
?>

File: Newclass.py
from flask import Flask, render_template, request, redirect
import sqlite3

app = Flask(Newtest.py)

@app.route('/create-class', methods=['POST'])
def create_class():
    class_name = request.form['class-name']
    class_period = request.form['period']
    
    conn = sqlite3.connect('Usertest.db')
    c = conn.cursor()
    c.execute("INSERT INTO classes (name, period) VALUES (?, ?)", (class_name, class_period))
    conn.commit()
    conn.close()

    return redirect('/classes.html')

@app.route('/')
def index():
    return render_template('Newclass.html')
